### PR TITLE
chore: release o3-tokkens and o3-buttons

### PR DESCRIPTION
**Pull request by akomiqaia**
## Describe your changes
Release o3-tokens and check ci/cd works for o3 components

### CODE REVIEW

File: components/o3-web-token/package.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
   "license": "MIT",
   "type": "module",
   "scripts": {
-    "build-tokens": "node ../../apps/dictionary/scripts/build-config/build.js"
+    "build-tokens": "node ../../apps/dictionary/scripts/build-config/build.js",
```
Comment (notlee) [inline components/o3-web-token/package.json:17]: Could this be `build`? Then we don't need to specify this in our Github Actions.

File: components/o3-web-token/package.json [hunk 2]
```diff
@@ -14,7 +14,8 @@
...
+		"lint": "bash ../../scripts/component/lint.bash"
```
Comment (notlee) [inline components/o3-web-token/package.json:18]: As discussed with o3-button, let's come back to linting

File: components/o3-web-token/package.json (hunk 1 above)
Comment (notlee) [reply components/o3-web-token/package.json:17]: We don't generate a build directory for o3-web-token. How does that affect my import statement as a user? Consistency would be good, to make components easier to use

Comment (notlee) [inline components/o3-web-token/package.json:17]: Like you did for 03-button (👏 ) perhaps we could also include `files`, to be explicit about what files should be installed

https://docs.npmjs.com/cli/v10/configuring-npm/package-json#files

File: components/o3-web-token/.gitignore [hunk 3]
```diff
@@ -3,7 +3,6 @@
 /.sass-cache/
 /bower_components/
 /node_modules/
-/build/
```
Comment (notlee) [inline components/o3-web-token/.gitignore:6]: Files are built on deploy, but never committed. Therefore the build files that are committed in this PR will become out of date. I think we should continue to `.gitignore` build files. Is there are reason you removed this?

File: components/o3-web-token/README.md [hunk 4]
```diff
@@ -1,56 +1,35 @@
...
-	- [Contributing](#contributing)
 	- [Migration](#migration)
 	- [Contact](#contact)
 	- [Licence](#licence)
 
-_Instructions for the component. We've broken this down by Markup, JavaScript, Sass, etc. You may add more sections as needed._
 
 ## Usage
 
+Check out [how to include Origami components in your project](https://origami.ft.com/documentation/components/#including-origami-components-in-your-project) to get started with `o3-web-token`.
```
Comment (notlee) [inline components/o3-web-token/README.md:12]: This line doesn't apply to o3 – we don't have global o3 tutorials/documentation yet. Let's remove this line (I wrote this comment before but maybe forgot to post ❓ )

File: components/o3-web-token/.gitignore (hunk 3 above)
Comment (akomiqaia) [reply components/o3-web-token/.gitignore:6]: yeah, but these are tokens for brands right? and when we commit tokens from Figma we want to know what changed in the code as well. I was a bit hesitant to put brands in build but for consistency made sense

Comment (notlee) [reply components/o3-web-token/.gitignore:6]: Hmmmmmm... good point, I think probably so 👍 We can add automation later to build and commit when token changes come in from Token Studio

**Review by notlee — APPROVED** [review]

**Pull request merged**